js/task-tracker.js

```diff
@@ -30,16 +30,27 @@ const themes = {
     purple: {
         color: "#571491",
         background: "#EEE5F6"
+    },
+    low: {
+        color: "#3B6528",
+        background: "#DEFCD0"
+    },
+    medium: {
+        color: "#571491",
+        background: "#EEE5F6"
+    },
+    high: {
+        color: "#C15751",
+        background: "#E9D4D3"
     }
 };
```

Comment:
Please, extract it in module and import or require it to use in other places

Reply:
I made this a static property of the class `Task`, because it will use it correctly at the class level, not one specific object